include/deal.II/lac/arpack_solver.h

```diff
@@ -238,6 +246,17 @@ class ArpackSolver : public Subscriptor
   bool initial_vector_provided;
   std::vector<double> resid;
 
+  /**
+   * Real part of the shift
+   */
+  double sigmar;
```

Comment:
that's how it is [used internally in Arpack](https://github.com/dealii/dealii/blob/master/include/deal.II/lac/arpack_solver.h#L614-L626), so i would not do it here.